### ISSUE TITLE
[dataio] filestager test broken by python 2.7.9 ssl change (Trac #1054)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1048]
On python 2.7.9, I get: `urlopen error [SSL: CERTIFICATE_VERIFY_FAILED]`

This is a result of the ssl security features added in https://www.python.org/dev/peps/pep-0476, and our use of a self-signed cert for testing.

This is a placeholder ticket to notify people that it's already being worked on.

Migrated from https://code.icecube.wisc.edu/ticket/1054
```json
{
    "status": "closed", 
    "changetime": "2019-01-11T21:45:42", 
    "description": "On python 2.7.9, I get: `urlopen error [SSL: CERTIFICATE_VERIFY_FAILED]`\n\nThis is a result of the ssl security features added in https://www.python.org/dev/peps/pep-0476, and our use of a self-signed cert for testing.\n\nThis is a placeholder ticket to notify people that it's already being worked on.\n\n", 
    "reporter": "david.schultz", 
    "cc": "jvansanten", 
    "resolution": "fixed", 
    "_ts": "1547243142967943", 
    "component": "combo core", 
    "summary": "[dataio] filestager test broken by python 2.7.9 ssl change", 
    "priority": "major", 
    "keywords": "", 
    "time": "2015-07-15T22:05:40", 
    "milestone": "", 
    "owner": "david.schultz", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150715T22:06:02: david.schultz changed status from "new" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20150715T22:17:31: 
* david.schultz commented:

> In changeset 2163:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="2163"
> fixes #1048, though not in the best way. left a TODO describing a better way.
> ```


* david.schultz changed resolution from "" to "fixed"
* david.schultz changed status from "accepted" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20150810T20:31:17: 
* olivas changed milestone from "offline V15-07-00" to ""
* olivas commented:

> Milestone offline V15-07-00 deleted

**Comment by icecube-trac** [comment]
Trac update at 20190111T21:45:42: nwhitehorn commented:

> In changeset 169143:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="169143"
> fixes #1048, though not in the best way. left a TODO describing a better way.
> ```
